cli/release.go

```diff
@@ -63,6 +81,37 @@ func runRelease(args *docopt.Args, client *controller.Client) error {
 	return fmt.Errorf("Top-level command not implemented.")
 }
 
+func runReleaseShow(args *docopt.Args, client *controller.Client) error {
+	var release *ct.Release
+	var err error
+	if args.String["<id>"] != "" {
+		release, err = client.GetRelease(args.String["<id>"])
+	} else {
+		release, err = client.GetAppRelease(mustApp())
+	}
+	if err != nil {
+		return err
+	}
+	artifact, err := client.GetArtifact(release.ArtifactID)
+	if err != nil {
+		return err
+	}
+	types := make([]string, 0, len(release.Processes))
+	for typ := range release.Processes {
+		types = append(types, typ)
+	}
+	w := tabwriter.NewWriter(os.Stdout, 1, 2, 2, ' ', 0)
+	defer w.Flush()
+	listRec(w, "ID", release.ID)
+	listRec(w, "Artifact", fmt.Sprintf("%s+%s", artifact.Type, artifact.URI))
+	listRec(w, "ProcessTypes", strings.Join(types, ","))
+	listRec(w, "CreatedAt", release.CreatedAt)
```

Comment:
Hmm so I always have shell filtering in the back of my mind, so the first column having no spaces and no extraneous colon helps in that regard.

That being said, I could probably be persuaded...